spec/spec.html

```diff
@@ -0,0 +1,25 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <title>Mocha</title>
+    <meta http-equiv="Content-Type" content="text/html; charset=UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="../node_modules/mocha/mocha.css" />
+  </head>
+  <body>
+    <div id="mocha"></div>
+    <script src="../node_modules/mocha/mocha.js"></script>
+    <script src="../node_modules/chai/chai.js"></script>
+    <script src="../node_modules/knockout/build/output/knockout-latest.js"></script>
+    <!-- Application code goes here -->
+    <script src="../calculator.js"></script>
+
+    <script>mocha.setup('bdd');</script>
```

Comment:
Tell mocha to load `describe`, `it`, `beforeEach`, etc into the global scope as that is what our tests use